Clarify how reported usage works for failed/canceled jobs (#1853)
Qiskit/documentation: docs/guides/execution-modes.mdx

## Sessions versus batch usage

> Usage is a measurement of the amount of time the QPU is locked for your workload. It is important to understand how usage is calculated because it impacts how much you are charged for jobs (including failed jobs).

Reviewer: ```suggestion
Usage is a measurement of the amount of time the QPU is locked for your workload. Failed or canceled jobs count toward your reported usage in certain circumstances - see the [Failed and canceled jobs](#failed-job) section for more details.
```

![This image shows multiple sets of jobs.  One set is being run in session mode and the other is being run in batch mode.  For session mode, between each job is the interactive TTL (time to live).  The active window starts when the first job starts and ends after the last job is completed. After the final job of the first set of jobs completes, the active window ends and the session is paused (but not closed).  Another set of jobs then starts and jobs continue in a similar manner. The QPU is reserved for your use during the entire session.  For batch mode, the classical computation part of each job happens simultaneously, then all jobs are sent to the QPU.  The QPU is locked for your use from the time the first job reaches the QPU until the last job is done processing.  There is no gap between jobs where the QPU is idle.](/images/guides/execution-modes/SessionVsBatch.svg 'Sessions compared to batch')

<span id="failed-job"></span>
## Failed and canceled jobs

When a job is failed or canceled, you are charged for usage as follows:

> * Job or batch mode: you are charged for all the time the QPU was locked for the execution, regardless of cancellations and errors. So if the error occurs before the lock, you are charged nothing. If the error/cancellation occurs after the system lock, you are charged based on the value that  _channels service_ returns as `consumed`. Thus, some jobs failed jobs have no cost and others do.

Reviewer: ```suggestion
* Job or batch mode: you are charged for all the time the QPU was locked for the execution, regardless of cancellations and errors. So if the error occurs before the lock, you are charged nothing. If the error/cancellation occurs after the system lock, you are charged based on the value that  _channels service_ returns as `consumed`. Thus, some failed jobs have no cost and others do.
```

> * Job or batch mode: you are charged for all the time the QPU was locked for the execution, regardless of cancellations and errors. So if the error occurs before the lock, you are charged nothing. If the error/cancellation occurs after the system lock, you are charged based on the value that  _channels service_ returns as `consumed`. Thus, some jobs failed jobs have no cost and others do.

Reviewer: Will _channels service_ be understood by readers?

> When a job is failed or canceled, you are charged for usage as follows:
> 
> * Job or batch mode: you are charged for all the time the QPU was locked for the execution, regardless of cancellations and errors. So if the error occurs before the lock, you are charged nothing. If the error/cancellation occurs after the system lock, you are charged based on the value that   Qiskit Runtime returns as `consumed`. Thus, some failed jobs have no cost and others do.
> 
> * Session mode: You are charged all the wall-clock time from the first job execution until the session goes inactive or is closed, regardless of numbers of jobs, cancellations, or errors.

Reviewer: ```suggestion
When a job is failed or canceled, your usage is reported as follows:

* Job or batch mode: usage is reported for all the time the QPU was locked for the execution, regardless of cancellations and errors. If the error occurs before the lock, no usage is reported. If the error/cancellation occurs after the system lock, usage is reported based on the value that Qiskit Runtime returns as `consumed`. Thus, some failed jobs do not appear in your reported usage and others do.

* Session mode: Usage is reported for all the wall-clock time from the first job execution until the session goes inactive or is closed, regardless of numbers of jobs, cancellations, or errors.
```
